pennylane_lightning/core/src/simulators/lightning_tensor/tncuda/tests/Tests_MPSTNCuda.cpp

```diff
@@ -178,4 +177,17 @@ TEMPLATE_TEST_CASE("MPSTNCuda::getDataVector()", "[MPSTNCuda]", float, double) {
         CHECK(expected_state ==
               Pennylane::Util::approx(mps_state.getDataVector()));
     }
+
+    SECTION("Throw error for getDataVector() on device") {
+        std::size_t num_qubits = 100;
```

Comment:
Would you add tests for zero number of qubits?

Reply:
MPS won't work for 0 and 1 qubit cases with cutensornet. This check is added.